.github/workflows/ubuntu.yaml

```diff
@@ -3,7 +3,6 @@ name: ubuntu
 on: [push, pull_request]
 
 jobs:
-
   ubuntu-build:
```

Comment:
can you please revert the change to this file?

Reply:
https://github.com/PX4/PlotJuggler/actions/runs/12758129605/job/35559747796
this is pre-commit corrections, not intentional. It fails without them